backend/spec/features/admin/orders/shipments_spec.rb

```diff
@@ -37,6 +37,15 @@
       expect(page).to have_content("shipped package")
       expect(order.reload.shipment_state).to eq("shipped")
     end
+
+    it "doesn't send a shipping confirmation email when ask to suppress the mailer" do
+      find("#send_mailer").set(false)
+      
+      expect {
+        find(".ship-shipment-button").click
+        wait_for_ajax
+      }.not_to change{ ActionMailer::Base.deliveries.count }
```

Comment:
We can wrap the test above this one (`it "can ship a completed order"`) with the opposite email expectation. You'll probably want to rebase first, since that spec was changed in #1668

```
it "can ship a completed order" do
  expect {
    find(".ship-shipment-button").click

    expect(page).to have_content("shipped package")
    expect(order.reload.shipment_state).to eq("shipped")
  }.to change{ ActionMailer::Base.deliveries.count }.by(1)
end
```